utils/docker-utils.sh

```diff
@@ -129,7 +138,82 @@ docker::prune_configs() {
     local -r CONFIG_LABEL=${1:?"FATAL: remove_configs_by_label CONFIG_LABEL not provided"}
 
     # shellcheck disable=SC2046
-    docker config rm $(docker config ls -qf label=name="$CONFIG_LABEL") &>/dev/null
+    if [[ -n $(docker config ls -qf label=name="$CONFIG_LABEL") ]]; then
+        log info "Waiting for configs to be removed..."
+
+        docker config rm $(docker config ls -qf label=name="$CONFIG_LABEL") &>/dev/null
+
+        overwrite "Waiting for configs to be removed... Done"
+    fi
+}
+
+# Deploy a service, it will set config digests (in case a config is defined in the compose file)
+#
+# Arguments:
+# - $1 : docker compose path, e.g. "/instant/monitoring" ...
+# - $2 : docker compose file, e.g. "docker-compose.yml" "docker-compose.cluster.yml" ...
+# - $3 : docker compose dev file, e.g. "docker-compose.dev.yml"
+# - $4 : services names, e.g. "monitoring" "hapi-fhir" ...
+#
+docker::deploy_service() {
+    local -r DOCKER_COMPOSE_PATH="${1:?"FATAL: function 'deploy_service' is missing a parameter"}"
+    local -r DOCKER_COMPOSE_FILE="${2:?"FATAL: function 'deploy_service' is missing a parameter"}"
+    local -r DOCKER_COMPOSE_DEV_FILE="${3:?"FATAL: function 'deploy_service' is missing a parameter"}"
```

Comment:
This should not be a compulsory variable, otherwise running in dev mode is compulsory

Reply:
Yes thanks, fixed in [11cd3ea](https://github.com/jembi/platform/pull/188/commits/11cd3ea621e407cef0783128485381b0972f527e)